src/tablib/formats/_xlsx.py

```diff
@@ -51,6 +57,11 @@ def export_set(cls, dataset, freeze_panes=True, invalid_char_subst="-"):
         )
 
         cls.dset_sheet(dataset, ws, freeze_panes=freeze_panes)
+        if isinstance(column_width, str) and column_width != "adaptive":
+            raise ValueError(f"Unsupported value `{column_width}` passed to `column_width` "
+                             "parameter. It supports 'adaptive' or integer values")
```

Comment:
Maybe something like this, to be closer to existing exceptions?

And let's start assigning exception messages to a variable first, for the reasons set out at https://github.com/henryiii/flake8-errmsg

```suggestion
            msg = (
                f"Invalid value for column_width: {column_width}. "
                f"Must be 'adaptive' or an integer."
            )
            raise ValueError(msg)
```